app/uk/gov/hmrc/cataloguefrontend/actions/UmpAuthenticated.scala

```diff
@@ -45,4 +48,7 @@ class UmpAuthenticated @Inject()(userManagementAuthConnector: UserManagementAuth
     }
   }
 
+  override def parser: BodyParser[AnyContent] = cc.parsers.defaultBodyParser
+
+  override protected def executionContext: ExecutionContext = cc.executionContext
```

Comment:
Here comes this a bit awkward and possibly confusing thing for people (effectively it's all about the problem of whether to use MdcLoggingExecutionContext or Slf4jLogger's Markers). Because of ActionBuilder we are forced to provide an execution context and we have to take it from ControllerComponents and at the same time, in the `invokeBlock` we use the implicit conversion from HeaderCarrier => MdcLoggingExecutionContext. Personally, I don't really like this ambiguity and I'm not sure if people won't get lost and understand what's really going on. What about we spend some time and think thoroughly how to tackle this issue.

Reply:
To be addressed as part of https://jira.tools.tax.service.gov.uk/browse/PLATOPS-1449